### PR TITLE
HDDS-2810. recipe/Prometheus.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc translation

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2810

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by iamabug** [comment]
retest

File: hadoop-hdds/docs/content/recipe/Prometheus.zh.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+[Prometheus](https://prometheus.io/)是在[云原生计算基金会](https://www.cncf.io/)管理下开发的一款开源监控软件。
+
+Ozone 原生支持 Prometheus，各个组件会启动一个兼容 Prometheus 的指标 endpoint，将所有可用的 hadoop 指标以 Prometheus 的格式导出发布。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/recipe/Prometheus.zh.md:25]: endpoint->端点

Comment (smengcl) [reply hadoop-hdds/docs/content/recipe/Prometheus.zh.md]: done in 022222a06641b529642737159b135ed837f5b58c

File: hadoop-hdds/docs/content/recipe/Prometheus.zh.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  limitations under the License.
+-->
+
+[Prometheus](https://prometheus.io/)是在[云原生计算基金会](https://www.cncf.io/)管理下开发的一款开源监控软件。
+
+Ozone 原生支持 Prometheus，各个组件会启动一个兼容 Prometheus 的指标端点，将所有可用的 hadoop 指标以 Prometheus 的格式导出发布。
+
+## 准备
+
+ 1. [安装启动]({{< ref "start/RunningViaDocker.md" >}})一个 Ozone 集群。
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/recipe/Prometheus.zh.md:29]: Causes compile error:

```
REF_NOT_FOUND: Ref "start/RunningViaDocker.md": "/github/workspace/hadoop-hdds/docs/content/recipe/Prometheus.zh.md:29:20": page not found
```

```suggestion
 1. [安装启动]({{< ref "start/RunningViaDocker.zh.md" >}})一个 Ozone 集群。
```

**Comment by adoroszlai** [comment]
Thanks @iamabug for the translation, @xiaoyuyao and @smengcl for the reviews.

**Pull request merged**